server/gokbg3/grails-app/domain/org/gokb/cred/TitleInstancePackagePlatform.groovy

```diff
@@ -1575,4 +1578,65 @@ class TitleInstancePackagePlatform extends KBComponent {
 
   }
 
+  @Transient
+  static TitleInstancePackagePlatform getCorrectTipp(ArrayList<TitleInstancePackagePlatform> tipps = null, String url = null, JSONArray identifiers = null, Package aPackage = null){
+
+    //println("findCorrectTipp")
+    //println(tipps)
+    //println(url)
+    //println(identifiers)
+    //println(aPackage)
+
+    List tippsCorrect
+
+    //URL check
+    tippsCorrect = tipps.findAll {it.url == url}
+
+    if(tippsCorrect.size() == 1){
```

Comment:
Can we presume at this place that there is always at least one URL which matches?